packages/data/README.md

```diff
@@ -332,6 +332,9 @@ _Returns_
 Given the name of a registered store, returns an object of the store's action creators.
 Calling an action creator will cause it to be dispatched, updating the state value accordingly.
 
+Note: If the action creator is a generator or returns a promise, a promise is
```

Comment:
Needs to be updated given latest revisions?

Reply:
yup, forgot I manually added this.